src/widgets/course-main/course-main.tsx

```diff
@@ -43,7 +43,7 @@ export const CourseMain = ({ courseName, lang = 'en', type }: CourseMainProps) =
       <div className={`content ${styles.content}`}>
         <Image className={styles.icon} src={secondaryIcon} alt={title} lazy="false" />
         <div className={styles.info}>
-          <SectionLabel label={status} />
+          <SectionLabel fontSize="small" marginSize="small" label={status} />
```

Comment:
This and other
```suggestion
          <SectionLabel label={status} />
```

Reply:
Or 
```suggestion
          <SectionLabel>{status}</SectionLabel>
```
if we decided to use children.